README.md

```diff
@@ -264,15 +264,70 @@ Update coming...
 |`lowest_coverage_lfs(n=10)` | Show the `n` LFs with lowest coverage |
 |`lowest_empirical_accuracy_lfs(n=10)` | Show the `n` LFs with the lowest empirical accuracy against ground truth for candidates in the devset |
 |`lf_summary_table()` | Print a table with summary statistics for each LF |
-|`learn_weights(n_iter=1000, tol=1e-6, sample=False, n_samples=100, mu=None, n_mu=20, mu_min_ratio=1e-6, alpha=0, rate=0.01, decay=0.99, bias=False, warm_starts=False, use_sparse=True, verbose=False, log=True, plot=True)` | Use data programming to learn an elastic net logistic regression on candidates not in training set <ul><li>`n_iter`: maximum number of SGD iterations</li><li>`tol`: tolerance for relative gradient magnitude to weights for convergence</li><li>`sample`: Use batch SGD</li> <li>`n_samples`: Batch size for SGD</li><li>`mu`: sequence of regularization parameters to fit; if `None` and validation set exists, automatic sequence is chosen s.t. largest value is close to minimum value at which all weights are zero; if `None` and no validation set exists, uses a default parameter</li><li>`n_mu`: number of regularization parameters for automatic sequence</li><li>`mu_min_ratio`: lower bound for regularization parameter automatic sequence as ratio to largest value</li><li>`alpha`: elastic net mixing parameter (0 for l<sub>2</sub>, 1 for l<sub>1</sub>)</li><li>`rate`: SGD learning rate</li><li>`decay`: SGD learning rate decay</li><li>`bias`: use bias term</li><li>`warm_starts`: use warm starts when fitting a sequence of mu values</li><li>`use_sparse`: use sparse operations</li><li>`verbose`: be verbose</li><li>`log`: log result in `ModelLogger`</li><li>`plot`: show diagnostic plot for validation set performance</li></ul> |
+|`learn_weights(**kwargs)` | See [Learning in DDLite](#learnopts) below |
 |`learn_weights_validated(**kwargs)` | Wrapper for `learn_weights()`|
 |`set_use_lfs(use=True)` | Setter for using LF weights in prediction |
 |`get_log_odds(subset=None)` | Get log odds values for all candidates (`subset=None`), an index subset (`subset`), test set (`subset='test'`), or validation set (`subset='validation'`) |
 |`get_predicted_probability(subset=None)` | Get predicted probabilities for candidates |
 |`get_predicted(subset=None)` | Get predicted responses for candidates |
+|`get_log_odds_lf(subset=None)` | Get log odds values for all candidates (`subset=None`), an index subset (`subset`), test set (`subset='test'`), or validation set (`subset='validation'`) using only LFs|
```

Comment:
?  Not clear what this is?  Is this the weights of the LFs, or the marginals for the candidates, or...?